arb_mat.rst

```diff
@@ -0,0 +1,759 @@
+.. _arb-mat:
```

Comment:
This file should probably not be here?

Reply:
Dang, thanks for catching this

Reply:
At first I thought you had started to add documentation, which would have been great 😂